python_requires

```diff
@@ -18,32 +18,37 @@ import argparse
 import glob
 import os
 import re
-import tarfile
 
+from metaextract import utils as meta_utils
 from packaging.requirements import Requirement
 
 
-def read_from_tarball(tar_name, filename):
-    with tarfile.open(tar_name) as tar:
-        try:
-            file_found = min((f for f in tar.getnames()
-                              if f.endswith(filename)),
-                             key=len)
-        except ValueError:
-            raise Exception("Coud not find a file %s in %s" %
-                            (filename, tar_name))
-        t = tar.getmember(file_found)
-        file_content = tar.extractfile(t).read()
+def _requires_get_source(all_requires, pkg):
+    if pkg in all_requires['install']:
+        return 'install'
+    elif pkg in all_requires['extras']:
+        return 'extras'
+    elif pkg in all_requires['tests']:
+        return 'tests'
+    else:
+        return 'unknown'
+
```

Comment:
Hm. I don't expect something different than the 3 keys. So I would leave them explicit here